Courses/Navigation/NavigationBar.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  NavigationBar.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 11.10.2022.
+//
+
+import SwiftUI
+
+struct NavigationBar: View {
+    var title = ""
+    @Binding var hasScrolled: Bool
```

Comment:
The same is here. `NavigationBar` has collapsed and expanded states, but not `hasScrolled`

Reply:
@yura-hb , so problem only in name of variable, right?

Reply:
In general, yes. But when you create some object, then it is better to hide its internals. Otherwise, anybody can start to build logic depending on them. In case, you want later to refactor view. You need refactor it everywhere

Reply:
@yura-hb , ok thx!

Reply:
Just checking there is no misunderstanding that this is a custom NavigationBar. It detects if any view was scrolled and then changes a few of its properties based on that. So after the user scrolls any View, hasScrolled changes to true and this Navigation Bar changes in opacity, blur and font size etc.

Reply:
Thought about this and actually makes sense that in the context of Navigation bar, hasScrolled doesn't make sense as the navigation bar shouldn't care if user scrolled  another view or not, it should only care if it's collapsed or not. I will be updating this on my end as well.